src/main/java/jewellery/inventory/service/ProductService.java

```diff
@@ -26,12 +26,17 @@
 import jewellery.inventory.model.resource.ResourceInProduct;
 import jewellery.inventory.repository.*;
 import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @RequiredArgsConstructor
 public class ProductService implements EntityFetcher {
+  private static final Logger logger = LogManager.getLogger(ProductService.class);
+  private static final String NEW_OWNER_ID = "}. New owner with ID: {";
+  private static final String PRODUCT_ID = "}, Product ID: {";
```

Comment:
let's not use these constants. They seem really odd. I think they make the log message harder to understand in this case